tests/test_tutorial/test_request_files/test_tutorial001.py

```diff
@@ -152,35 +150,35 @@ def test_post_body_json():
     assert response.json() == file_required
 
 
-def test_post_file(tmpdir):
-    path = os.path.join(tmpdir, "test.txt")
-    with open(path, "wb") as file:
-        file.write(b"<file content>")
+def test_post_file(tmp_path):
+    path = tmp_path / "test.txt"
+    path.write_bytes(b"<file content>")
```

Comment:
I love these refactors using `pathlib` with `tmp_path`, thanks for that! :rocket: